common/src/main/java/io/netty/util/HashedWheelTimer.java

```diff
@@ -567,11 +565,6 @@ public boolean isExpired() {
             return state() == ST_EXPIRED;
         }
 
```

Comment:
will do... Also I can simplify this code a bit now that we not use MpscLinkedQueueNode anymore :)